tools/run_local_http_server.py

```diff
@@ -36,3 +37,7 @@
   - hangs forever: {local_server_http.url_hang_forever()}
   - bad ssl: {local_server_bad_ssl.url_200()}
 """)
+
+while True:
```

Comment:
curious why is it needed?

Reply:
the script is expected to be run manually. Probably there is a more elegant solution, but this simply keeps the main thread running in order to keep the thread with the http server running as well.